src/content/en/updates/2018/03/present-web-pages-to-secondary-attached-displays.md

```diff
@@ -0,0 +1,268 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Present web pages that support the Presentation API to a secondary attached displays.
+
+{# wf_updated_on: 2018-03-27 #}
+{# wf_published_on: 2018-03-27 #}
+{# wf_tags: news,media,presentation #}
+{# wf_featured_image: /web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/hero.jpg #}
+{# wf_featured_snippet: Chrome 66 allows users to present web pages that support the Presentation API to a secondary attached display, and control the contents being shown from the controlling page on the main display with the Presentation Receiver API. #}
+{# wf_blink_components: Blink>PresentationAPI #}
+
+# Present Web Pages to Secondary Attached Displays {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome 66 allows users to present web pages that support the [Presentation
+API] to a secondary attached display, and control the contents being shown from
+the controlling page on the main display with the [Presentation Receiver API].
+
+<figure>
+  <img src="/web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/step1.jpg"
+    alt="Step 1"/>
+  <figcaption style="text-align: center">
+    1/2. User picks a secondary attached display
+  </figcaption>
+</figure>
+<figure>
+  <img src="/web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/step2.jpg"
+    alt="Step 2"/>
+  <figcaption style="text-align: center">
+    2/2. A web page is automatically presented to the display previously picked
+  </figcaption>
+</figure>
+
+## Background
+
+Until now, web developers could build experiences where user would see some
+local content in Chrome that is different from the content they’d see on a
+remote display, while still being able to control that experience locally.
+We’re talking managing a playback queue on youtube.com for instance while
+videos play on the TV, or seeing a slide reel with speaker notes on laptop
+while the fullscreen presentation is shown in a Hangout session.
+
+There are scenarios though where users may simply want to present content onto
+a second, attached display. For example, a user in a conference room outfitted
```

Comment:
This is not a complete sentence. The simplest fix:
'For example, a user' > 'For example, imagine a user'

Reply:
Done.